components/header/header.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import Image from 'next/image';
+
+import style from './header.module.scss';
+
+export default function Header() {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  return (
+    <div className={style.container}>
+      <div className={style.header}>
+        <a href="#" className={style.logo}>
+          <Image
+            alt="KedroLogo"
+            src="/images/kedro-logo.svg"
+            layout="intrinsic"
```

Comment:
If you feel like, we can remove this `layout="intrinsic"` prop since it's the default, though it's no problem to have it. Your call.

If you remove, you can do so from all other places in this file.